lib/mongoose_push_web/controllers/api_v1_controller.ex

```diff
@@ -0,0 +1,34 @@
+defmodule MongoosePushWeb.APIv1Controller do
+  alias MongoosePushWeb.Schemas
+  alias OpenApiSpex.Operation
+  use MongoosePushWeb, :controller
+
+  @spec open_api_operation(atom) :: Operation.t()
+  def open_api_operation(action) do
+    operation = String.to_existing_atom("#{action}_operation")
+    apply(__MODULE__, operation, [])
+  end
+
+  @spec handle_operation() :: Operation.t()
+  def handle_operation() do
+    %Operation{
+      tags: ["apiv1"],
+      summary: "sends a push",
+      description: "performs the sending of push notification",
+      operationId: "APIv1Controller.handle",
```

Comment:
This operation should not be called `handle`. In fact, no operations should be called that. This is not meaningful. Let's imagine we add second thing to the API, e.g. `/api/v1/batch_push`, how would it be called? 
IMHO, this one should be simply `push`.

Reply:
Or maybe `send_notification`?